.github/pull_request_template.md

```diff
@@ -15,6 +15,9 @@ For new extensions:
 
 - [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).
```

Comment:
How about adding the `extension version` to the `For new extensions:` section?
`- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)`

Reply:
Cause all existing modules need to comply this rule in future releases, so put it in general guidelines.